docs/triggers/scheduled-trigger.md

```diff
@@ -0,0 +1,201 @@
+
+Both Single Occurrence and Recurring Schedules have common base attributes:
+
+| Parameter name  | Required?     | Description                                                          |
+| ----------------|:-------------:| ---------------------------------------------------------------------|
+| `start_time`      | Yes  |An ISO 8601 date string of when this scheduled Trigger should start (i.e. "2022-03-01T14:00:00Z")|
+| `timezone`      | No  |A timezone string to use for scheduling (Defaults to UTC if left blank)|
+
+A Single occurrence schedule can be created using just these two parameters, however a recurring schedule may also include the following parameters: 
+
+| Parameter name  | Required?     | Description                                                          |
+| ----------------|:-------------:| ---------------------------------------------------------------------|
+| `end_time`      | No  |An ISO 8601 date string of when this scheduled Trigger should end (i.e. "2022-03-01T14:00:00Z")|
+| `occurrence_count`| No  |The maximum number of times the Trigger may run (number)|
+| `frequency`      | No  |The configurable frequency of when this Trigger will activate|
+
+The `frequency` object contains information on the repeating schedule for the Trigger, it contains the following parameters:
+
+| Parameter name  | Required?     | Description                                                          |
+| ----------------|:-------------:| ---------------------------------------------------------------------|
+| `type`      | Yes  |How often the Trigger will activate|
+| `on_days` | No  |An array of days of the week as capitalized strings i.e (["Monday", "Tuesday"]) the Trigger should activate on (not available for daily triggers)|
+| `repeats_every`      | No  |How often the Trigger will repeat, respective to the frequency type (a number, i.e every `3` days)|
+| `on_week_num`      | No  |The nth week of the month that a monthly schedule should activate (a number i.e. activate on the `2` week of the month|
+
+### Scheduled Inputs Object
+The Scheduled Trigger has a required `inputs` object which can take the following parameters:
+
+| Parameter name  | Required?     | Description                                                          |
+| ----------------|:-------------:| ---------------------------------------------------------------------|
+| `type`      | No  |	The type of Trigger: `scheduled`|
+| `team_id` | No  |	A unique identifier for the workspace/team where the Trigger occurred.|
+| `enterprise_id`      | No  |	A unique identifier for the enterprise where the Trigger occurred.|
+| `event_timestamp`      | No  |	A UNIX timestamp in seconds indicating when the Trigger occurred.|
+
+
```

Comment:
There is no special `inputs` object for Schedule Triggers that takes these parameters, there is just the common `inputs` field that takes the input parameters of the Workflow. If these values exist, they'd be part of the Data Context object mentioned below.